src/main/resources/google_checks.xml

```diff
@@ -93,7 +93,7 @@
             <property name="option" value="alone"/>
             <property name="tokens"
              value="CLASS_DEF, METHOD_DEF, CTOR_DEF, LITERAL_FOR, LITERAL_WHILE, STATIC_INIT,
-                    INSTANCE_INIT, ANNOTATION_DEF, ENUM_DEF"/>
+                    INSTANCE_INIT, ANNOTATION_DEF, ENUM_DEF, INTERFACE_DEF"/>
```

Comment:
> In google style, it says that it is okay to have something like interface I1 {} .... So shall I keep INTERFACE_DEF in the alone token list since others are there too or remove it?

this is really strange ... I am not sure why we use `alone`, even Google style clearly state that empty block is ok. Empty interfaces are very common pattern ([marker interfaces](https://www.baeldung.com/java-marker-interfaces)).

Reply:
It will be resolved in #7541 , in google config we use XpathSuppression:
https://github.com/checkstyle/checkstyle/blob/4768470f54ec6f3031bd335157db9413c3906be9/src/main/resources/google_checks.xml#L91-L102